Core/GDCore/Project/ExtensionPropertiesManager.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * GDevelop Core
+ * Copyright 2008-2016 Florian Rival (Florian.Rival@gmail.com). All rights
+ * reserved. This project is released under the MIT License.
+ */
+#ifndef GDCORE_EXTENSIONPROPERTIESMANAGERS_H
+#define GDCORE_EXTENSIONPROPERTIESMANAGERS_H
+#include <map>
+#include "GDCore/Serialization/SerializerElement.h"
+#include "GDCore/String.h"
+
+namespace gd{
+  class GD_CORE_API ExtensionPropertiesManager {
+    public:
+  	  gd::String& GetValue(const gd::String& extension, const gd::String& property) {
```

Comment:
I think you can remove this non const method - callers that want to modify a value will use SetValue ;)